vobject/vcard.py

```diff
@@ -147,7 +149,7 @@ def encode(cls, line):
         if not line.encoded:
             encoding = getattr(line, 'encoding_param', None)
             if encoding and encoding.upper() == cls.base64string:
-                line.value = line.value.encode('base64').replace('\n', '')
+                line.value = codecs.encode(line.value.encode(coding), "base64").decode("utf-8")
```

Comment:
Perhaps my pull request on top will fix this: https://github.com/eventable/vobject/pull/49/files